### ISSUE TITLE
Fusionscharts inside Angular 8.2.3 with angular-fusion 

### ISSUE THREAD

**Issue by rbinsztock** [fusioncharts/fusioncharts-dist#7]
Hi, we just updated from Angular 8.0 to Angular 8.2.3, so I bumped angular-fusioncharts and fusioncharts libs with it but I have a loading problem with zonejs.

```
main.ts:12 Error: Zone.js has detected that ZoneAwarePromise `(window|global).Promise` has been overwritten.
Most likely cause is that a Promise polyfill has been loaded after Zone.js (Polyfilling Promise api is not necessary when zone.js is loaded. If you must load one, do so before loading zone.js.)
    at Function.push../node_modules/zone.js/dist/zone.js.Zone.assertZonePatched (zone.js:58)
    at new NgZone (ng_zone.ts:123)
    at getNgZone (application_ref.ts:358)
    at PlatformRef.push../node_modules/@angular/core/fesm5/core.js.PlatformRef.bootstrapModuleFactory (application_ref.ts:255)
    at application_ref.ts:308
    at fusioncharts.js:13
```

- "angular-fusioncharts": "3.0.3",
- "fusioncharts": "3.14.0",
- "@angular/animations": "~8.2.3",
- "@angular/cdk": "^8.0.0",
- "@angular/common": "~8.2.3",
- "@angular/compiler": "~8.2.3",
- "@angular/core": "~8.2.3",
- "@angular/forms": "~8.2.3",
- "zone.js": "~0.9.1"

If I downgrade fusioncharts to 3.13.4 the zonejs error problem disappear but I still have an a warning issue about fusionscharts.s module : 

```
WARNING in ./node_modules/fusioncharts/themes/fusioncharts.theme.fusion.js
Module Warning (from ./node_modules/source-map-loader/index.js):
(Emitted value instead of an instance of Error) Cannot find SourceMap 'fusioncharts.theme.fusion.js.map': Error: Can't resolve './fusioncharts.theme.fusion.js.map' in 'C:\Users\zzzz\yyy\xxxx\node_modules\fusioncharts\themes'

WARNING in ./node_modules/fusioncharts/fusioncharts.charts.js
Module Warning (from ./node_modules/source-map-loader/index.js):
(Emitted value instead of an instance of Error) Cannot find SourceMap 'http://localhost:3052/3.13.4/map/eval/fusioncharts.charts.js.map': Error: Can't resolve './http://localhost:3052/3.13.4/map/eval/fusioncharts.charts.js.map' in 'C:\Users\zzzz\yyy\xxxx\node_modules\fusioncharts'

WARNING in ./node_modules/fusioncharts/fusioncharts.js
Module Warning (from ./node_modules/source-map-loader/index.js):
(Emitted value instead of an instance of Error) Cannot find SourceMap 'http://localhost:3052/3.13.4/map/eval/fusioncharts.common.js.map': Error: Can't resolve './http://localhost:3052/3.13.4/map/eval/fusioncharts.common.js.map' in 'C:\Users\zzzz\yyy\xxxx\node_modules\fusioncharts'
```

it's just a npm warning.

**Comment by ayanbhaduryfc** [comment]
The issue is fixed in version 3.14.0-sr.1, refer this link -https://github.com/fusioncharts/angular-fusioncharts/issues/61

FusionCharts also released a new version today 3.14.1, you can upgrade to the latest version to fix the problem - https://www.npmjs.com/package/fusioncharts

**Comment by rbinsztock** [comment]
The error with Zone.js is resolved with the last version (3.14.1), the npm warning is still here in console.

**Comment by AyanBhadury** [comment]
@rbinsztock Please share the warning you are getting
